cypress/integration/pages/frontPage/tests.js

```diff
@@ -134,6 +134,29 @@ export const testsThatFollowSmokeTestConfig = ({ service, pageType }) =>
             },
           );
         });
+        it('should contain Useful Links if usefulLinks block data exists', () => {
+          cy.request(`${config[service].pageTypes.frontPage.path}.json`).then(
+            ({ body }) => {
+              const pageData = body.content.groups;
+              const usefulLinks = pageData.find(data => {
+                return data.type === 'useful-links';
+              });
+              const hasStrapline = usefulLinks
+                ? Object.prototype.hasOwnProperty.call(usefulLinks, 'strapline')
+	const hasStrapline = usefulLinks && Object.prototype.hasOwnProperty.call(usefulLinks, 'strapline')
```

Comment:
github is stupid so the suggestion thing didn't really work. Replace lines 144&145 with my suggestion on line 146. Oh and don't forget about your semi-colons 😄